test/lib/next-modes/next-dev.ts

```diff
@@ -27,7 +28,7 @@ export class NextDevInstance extends NextInstance {
     let startArgs = [
       'yarn',
       'next',
-      useTurbo ? '--turbo' : undefined,
+      useTurbo ? getTurbopackFlag() : undefined,
```

Comment:
```suggestion
```

Reply:
why we have to take this line?

Reply:
going over codebase I think this is better to remain, it explicitly passes flag so we can check log outputs easier, as well as `getTurbopackFlag` throws for unexpected cases test trying to run with mismatching env.